### ISSUE TITLE
Latest font review, after Vietnamese and Kerning fixes. 

### ISSUE THREAD

**Issue by vv-monsalve** [xotypeco/big_shoulders#19]
Here are some comparative gifs of the font in this order:
    Frame 1: the font already published on Google Fonts
    Frame 2: the font on QA branch, before the last changes
    Frame 3: the latest file from the master branch.

While the file has now the same amount of kerning cases and the common ones (e.g., To) are better now, there are other changes that seem not necessary and could be the cause of reflows on text lines. (e.g., the tighter space between 'u' and 'e' or the looser space between 'o''o')

Please check the consistency and coherence of the spacing and kerning on the entire system (similar forms to share the same values, and the range of some variations) to avoid as much as possible the reflow.

---

![BigShoulders-Text](https://user-images.githubusercontent.com/48698976/84562368-a313e880-ad19-11ea-8415-2b610a95dfd1.gif)

----

![BigShoulders-Layout](https://user-images.githubusercontent.com/48698976/84562434-29302f00-ad1a-11ea-9c16-5041d332e301.gif)

----

![BigShoulders-Text-words](https://user-images.githubusercontent.com/48698976/84562334-5af4c600-ad19-11ea-8f36-4c5905223812.gif)

----

![BigShoulders-Display-words](https://user-images.githubusercontent.com/48698976/84562455-4fee6580-ad1a-11ea-9b28-bbaabd57aae7.gif)

**Comment by xotypeco** [comment]
fixed and re-saved. the main culprits were lowercase t followed by a space or a dash, and the oo kerning pair, both of which I'm almost positive I deleted by mistake.

this should fix everything. first frame is current version on GF, second frame is new.

once we're done here I’ll port all the metrics to Stencil and Inline and update those.
![old_new](https://user-images.githubusercontent.com/52611676/84576268-02551580-ad79-11ea-98d6-2a780b26f9b1.gif)

**Comment by xotypeco** [comment]
please hold on this. I just introduced a kerning pair into all but 2 of the masters, need to find it. will post when I do.

**Comment by xotypeco** [comment]
found & fixed it.

ok. as for re-flow, I'm not worried about this in the heavier weights. lighter weights, it makes sense to worry about, and those are fixed.

**Comment by xotypeco** [comment]
Big Shoulders proper and stencil are now re-committed to master. in-line is in progress.

the reason for all this reflow was that I used Georg’s Kern-a-Lytics extension for Glyphs for the first time to equalize kerning across masters. in doing so, I deleted some crucial kerns of t and r followed by a space. those are now fixed, so reflow in lighter weights should be no longer a problem. in heavier weights there *will* be some reflow, but I’m comfortable with the amount. it’s minimal.

in doing this I uncovered a lot of fixes that also needed to be made to Polish kerning—d, l & t followed by a caron needed a full re-kerning to complete a redesign i’d done a while ago.
